src/modules/common/components/ConditionalLink.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Link } from 'redux-little-router'
+import { branch, mapProps, renderComponent } from 'recompose'
+import { omit } from 'lodash/object'
+
+export class DisabledLink extends React.Component {
```

Comment:
I think we still mix up the words `dis-/enabled` and `(in)active`, don't we?
I think in this case DisabledLink is the right word (when a link is not even clickable).
An active Link is (for me) a link that indicates (visually) that the current site is the target of the link

Reply:
The problem about this is that we call "Extras" and "News" *enabled* if the locations really has it enabled via the cms. In my opinion that is the right word for it, too. But then if we also called the "News" Link *enabled/disabled* if there are news we would be ambiguous and have naming problems in HeaderNavigationItem. So atm, I call a HeaderNavigationItem active, if it should be shown normal (not grayed out), selected the target matches the current site and enabled if it is enabled via the cms. I agree that we mix the words, but maybe we should then call DisabledLink *InactiveLink*.

Reply:
Yes, that's a possible solution.
If you want you can also rename the ExtrasEnabled and NewsEnabled thingy to sth like ExtrasVisible and NewsVisible, maybe then we'll don't have ambigouities in any context (even though enabled may sound better than visible in the cms-context)... But that's up to you :D

Reply:
@klinzo @maxammann What is your opinion on that? :)

Reply:
I think extras/events enabled and selected should stay the way they are, enabled and selected are definitely the right terms for this. accessible would maybe be good imo for whether there are events to show or not

Reply:
I think I'm gonna stick to *active* then and rename DisabledLink